io.github.hydrusnetwork.hydrus.yml

```diff
@@ -0,0 +1,127 @@
+    config-opts:
+      - --enable-shared
+      - --disable-static
+      - --enable-gnutls
+      - --disable-doc
+      - --disable-ffprobe
+      - --disable-ffplay
+    sources:
+      - type: archive
+        url: https://ffmpeg.org/releases/ffmpeg-5.1.tar.gz
+        sha256: c0b2ec290abd5ad31b5e0a8c455c34f90306ecd2ede6ace5db5ca3a07f3b842f
+        x-checker-data:
+          type: anitya
+          project-id: 5405
+          stable-only: true
+          url-template: https://ffmpeg.org/releases/ffmpeg-$version.tar.gz
+    cleanup:
+      - /share/ffmpeg/examples
+
+  - name: libass
+    cleanup:
+      - /include
+      - /lib/*.la
+      - /lib/pkgconfig
+    config-opts:
+      - '--disable-static'
+    sources:
+      - type: archive
+        url: 'https://github.com/libass/libass/releases/download/0.15.0/libass-0.15.0.tar.gz'
+        sha256: '9cbddee5e8c87e43a5fe627a19cd2aa4c36552156eb4edcf6c5a30bd4934fe58'
+
+  - name: mpv
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /share/man
+    buildsystem: simple
+    build-commands:
+      - python3 waf configure --prefix=/app --enable-libmpv-shared --disable-cplayer
+      - python3 waf build
+      - python3 waf install
+    sources:
+      - type: archive
+        url: https://github.com/mpv-player/mpv/archive/refs/tags/v0.34.1.tar.gz
+        sha256: '32ded8c13b6398310fa27767378193dc1db6d78b006b70dbcbd3123a1445e746'
+      - type: file
+        url: 'https://waf.io/waf-2.0.21'
+        sha256: '7cebf2c5efe53cbb9a4b5bdc4b49ae90ecd64a8fce7a3222d58e591b58215306'
+        dest-filename: waf
+
+  - name: miniupnpc
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        sha256: 888fb0976ba61518276fe1eda988589c700a3f2a69d71089260d75562afd3687
+        url: https://miniupnp.tuxfamily.org/files/miniupnpc-2.2.2.tar.gz
+
+  - name: entrypoint
+    buildsystem: simple
+    build-commands:
+      - install -D hydrus.sh /app/bin/hydrus.sh
+    sources:
+      - type: file
+        path: hydrus.sh
+
+  - name: metadata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 io.github.hydrusnetwork.hydrus.desktop /app/share/applications/io.github.hydrusnetwork.hydrus.desktop
+      - install -Dm644 io.github.hydrusnetwork.hydrus.metainfo.xml /app/share/metainfo/io.github.hydrusnetwork.hydrus.metainfo.xml
+      - install -Dm644 io.github.hydrusnetwork.hydrus.128.png /app/share/icons/hicolor/128x128/apps/io.github.hydrusnetwork.hydrus.png
+      - install -Dm644 io.github.hydrusnetwork.hydrus.256.png /app/share/icons/hicolor/256x256/apps/io.github.hydrusnetwork.hydrus.png
+      - install -Dm644 io.github.hydrusnetwork.hydrus.512.png /app/share/icons/hicolor/512x512/apps/io.github.hydrusnetwork.hydrus.png
+    sources:
+      - type: file
+        path: io.github.hydrusnetwork.hydrus.128.png
+      - type: file
+        path: io.github.hydrusnetwork.hydrus.256.png
+      - type: file
+        path: io.github.hydrusnetwork.hydrus.512.png
```

Comment:
you could as well remove these two sizes are they are just magnified version of the other one and not necessary